src/features/payment/Payment.js

```diff
@@ -62,12 +63,16 @@ const Checkout = () => {
         },
       });
 
-      if (paymentContainer.current) {
+      if (paymentContainer.current && !ignore) {
```

Comment:
I'd like to understand why we the ignore property is needed here, please add a comment. Can we make it more elegant? It seems hacky 👍

Reply:
Proposed fix: maybe summarize it and link the article here in a comment
https://beta.reactjs.org/learn/synchronizing-with-effects#fetching-data

Reply:
Comment added!